### PR TITLE
Implemented Simple GridView to home screen (#55)

### PR DESCRIPTION
**Pull request by CloudyPadmal**
I've implemented Simple GridView to the home screen gallery. @Abhi2424shek @pa1pal can you see if this is ok? :)

- I've kept the main layout untouched because they seems to house a lot of other views programatically! So I'm shifting visibilities if the view is related to Local Photos!

- And there is a little problem... That is when I'm using camera, it saves the image in two places! So the gallery shows two same images. But if I click 'Discard' once the camera shows the taken picture, then it is saved in one location and the image is shown once!

homescreenwithgrid

### CODE REVIEW

**Comment by Abhi2424shek** [comment]
Can you open an issue for the second point pls, with more detailed info.

**Comment by CloudyPadmal** [comment]
Coming right away!

**Comment by Abhi2424shek** [comment]
Can you give an app, so that I could test it, diff seems fine

**Comment by CloudyPadmal** [comment]
Here's the APK! :) [app-debug.apk.zip](https://github.com/fossasia/phimpme-android/files/817283/app-debug.apk.zip)

File: app/src/main/java/vn/mbm/phimp/me/newGallery.java [hunk 1]
```diff
@@ -160,10 +163,17 @@
...
+    static ScrollView localPhotosScroll;
+    static GridView localPhotosGrid;
+    static PhotosAdapter photosAdapter;
+    static Activity localActivity;
+    static Cursor pathcursor;
     static TextView txtlocal_gallery;
     static GridView gv_local_gallery;
     static ImageButton btn_local_more;
     static int local_rows_display = 0;
+    static LinearLayout ln_local_gallery;
```
Comment (pa1pal) [inline app/src/main/java/vn/mbm/phimp/me/newGallery.java]: error: variable ln_local_gallery is already defined in class newGallery
See the line number-165

Comment (CloudyPadmal) [inline app/src/main/java/vn/mbm/phimp/me/newGallery.java]: Ahh I missed that! :O

Comment (CloudyPadmal) [inline app/src/main/java/vn/mbm/phimp/me/newGallery.java]: @pa1pal build successful :innocent:

**Comment by CloudyPadmal** [comment]
Thank you @pa1pal :medal_military:

**Pull request merged**